modules/local_contacts/modules.php

```diff
@@ -240,7 +240,7 @@ protected function output() {
             $button = '<input type="hidden" name="contact_id" value="'.$this->html_safe($current['id']).'" />'.
                 '<input class="btn btn-primary edit_contact_submit" type="submit" name="edit_contact" value="'.$this->trans('Update').'" />';
         }
-        return '<div class="add_contact kokokoko"><form class="" method="POST">'.
+        return '<div class="add_contact_responsive"><form class="" method="POST">'.
```

Comment:
Any value for class? in `<form class="" method="POST">`

Reply:
Hello @Baraka24  I just removed the `class` attribute. Please review

Reply:
I just added the attribute `class="add_contact_form"`